components/standings/commons/starcraft_starcraft2/standings_storage_starcraft_legacy.lua

```diff
@@ -25,6 +25,7 @@ local INVALID_OPPONENT_CATEGORY = '[[Category:Pages with invalid opponent '
 
 local Wrapper = {}
 
+---@param frame Frame
```

Comment:
Missing return

Reply:
it doesn't have a return value
it "only" stores in lpdb